packages/contracts/test/integration-tests/strategies/hack-money-strategy.ts

```diff
@@ -233,8 +233,8 @@ describe("Hack Money Strategy integration test", async () => {
       // const [collateralToAdd2] = await strategy.getRequiredCollateral(strikeObj2);
       // const collateralToAdd = collateralToAdd1.add(collateralToAdd2);
 
-      const { smallStrikePrice, bigStrikePrice } = await strategy.getStrikes();
-      console.log(smallStrikePrice.toString(), bigStrikePrice.toString());
+      // const { smallStrikePrice, bigStrikePrice } = await strategy.getStrikes();
```

Comment:
@Baraa42 I don't have such a method, so I had to comment this

Reply:
@igorline was just testing it selects strikes correctly, works ok